packages/web/src/pages/SignIn/__snapshots__/SignIn.test.tsx.snap

```diff
@@ -89,67 +89,72 @@ exports[`pages/SignIn should render successfully 1`] = `
       </div>
     </nav>
   </header>
-  <form
-    class=""
+  <div
+    class="container signInConrollersContainer"
```

Comment:
Same here. 
And just a question, do we use camelCase naming for CSS classes?

Reply:
I don't see what is wrong with camelCase in CSS